shared/cfg/base.cfg

```diff
@@ -832,6 +832,8 @@ nvdimm_uuid = ""
 # tpm_device_path = "/dev/tpm0"
 # tpm_model = "tpm-spapr"
 # tpm_type = "passthrough"
+# tpm_version = "1.2" # only qemu command line
```

Comment:
`only for qemu command line` or `qemu command line only`

Reply:
Hi @vivianQizhu
Thanks, have updated, please help me to review, thanks.